### ISSUE TITLE
Fix integration test "Interaction in autoprint.pdf must check if printing is triggered when the document is open"

### ISSUE THREAD

**Issue by timvandermeij** [mozilla/pdf.js#18382]
The most recent intermittent failure is observed in http://54.193.163.58:8877/b31ab06e42d0bd2/output.txt (on Windows, in PR #18380), but we have seen this one more often. The traceback is:

```
Failures:
1) Interaction in autoprint.pdf must check if printing is triggered when the document is open
  Message:
[31m    ProtocolError: Runtime.callFunctionOn timed out. Increase the 'protocolTimeout' setting in launch/connect calls for a higher timeout if needed.[0m
  Stack:
        at <instance_members_initializer> (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/common/CallbackRegistry.js:89:14)
        at new Callback (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/common/CallbackRegistry.js:93:16)
        at CallbackRegistry.create (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/common/CallbackRegistry.js:19:26)
        at Connection._rawSend (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/cdp/Connection.js:86:26)
        at CdpCDPSession.send (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/cdp/CDPSession.js:63:33)
        at #evaluate (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/cdp/ExecutionContext.js:353:50)
        at async ExecutionContext.evaluate (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/cdp/ExecutionContext.js:268:16)
        at async IsolatedWorld.evaluate (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/cdp/IsolatedWorld.js:96:16)
        at async CdpJSHandle.evaluate (file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/node_modules/puppeteer-core/lib/esm/puppeteer/api/JSHandle.js:139:20)
        at async file:///C:/pdfjs/botio-files-pdfjs/private/b31ab06e42d0bd2/test/integration/scripting_spec.mjs:1829:11
```

As far as I can tell this should be fixed by the print test changes in #17962, so given that that PR is not yet ready I think we should extract those changes and land them separately to piece-by-piece get rid of the remaining intermittents we have. Moreover, as an extra bonus this will enable a currently skipped test on Windows.

@calixteman Would you be willing to extract those print test changes into a separate PR?

**Comment by timvandermeij** [comment]
This problem is likely fixed by #18383, but due to its intermittent nature we'll leave this issue open for a bit to make sure it doesn't reappear.

**Comment by timvandermeij** [comment]
Closing since the biggest problems are fixed now and the remainder is tracked in #18396.
